### DISCUSSION TITLE
Just a thanks

### DISCUSSION THREAD

**Discussion by MatthewJMullins** [reuseman/flashcards-obsidian#52] [General]
Just wanted to say thanks @reuseman. It's not easy working for free, so a little appreciation goes a long way. I'm not a coder, but if I could contribute in some way, don't hesitate. 

## I noticed some advantages of this plugin after trying the other plugins:

- Note Simplicity - Using what obsidian already has: headings. Remnote style in-lines.

- Setup Simplicity - All card types are added automatically.

- Clean context aware - No extra setting up to create a field for the context. It adds it right to the front side. Reduces the amount of typing that would be needed for context.

- Back links to the note - This might be the coolest. Click and go to any linked notes, not just the source note.

## Have you considered...

- Dropping the last element of the "context" onto a newline? The question would be visually more identifiable from the context itself.

- Allowing multiple tags for "Flashcards #Tag" setting? This would allow user to get use within obsidian from #card, #term, #question, etc. while using them to produce cards.

**Comment by reuseman** [comment]
Thanks for the kind words.

About your proposal:
- The context thing, it's actually a good idea from a visual standpoint.
- The multiple tags for me seems just a way to have more cognitive overload in terms of usage. In terms of development, it's just too much customization and edge cases.
